inc/functions.php

```diff
@@ -1206,16 +1206,22 @@ function fetch_page_url($url, $page)
  * @param int $uid The user ID
```

Comment:
Could you update this comment to mention that passing `null` will use the current user's ID?

Reply:
Done.